deployments/README.md

```diff
@@ -79,8 +79,49 @@ not strictly required.
 
 ## General Notes
 
-Substrate authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`, `Ferdie`.
-Ethereum authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Rialto authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+Rialto-PoA authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Millau authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+
+Both authorities and following accounts have enough funds on corresponding Substrate chains:
+
+- on Rialto: `Ferdie`, `George`, `Harry`.
+- on Millau: `Ferdie`, `George`, `Harry`.
+
+Names of accounts on Substrate (Rialto and Millau) chains may be prefixed with `//` and used as
+seeds for the `sr25519` keys. This seed may also be used in the signer argument in substrate
+and PoA relays. Example:
+
+```bash
+./substrate-relay rialto-headers-to-millau \
+	--rialto-host rialto-node-alice \
+	--rialto-port 9944 \
+	--millau-host millau-node-alexander \
+	--millau-port 9944 \
+	--rialto-signer //Harry \
+	--prometheus-host=0.0.0.0
+```
+
+Some accounts are used by bridge components. Using these accounts to sign other transactions
+is not recommended, because this may lead to nonces conflict.
+
+Following accounts are used when `poa-rialto` bridge is running:
+
+- Rialto' `Alice` signs relay transactions with new Rialto-PoA headers;
+- Rialto' `Bob` signs relay transactions with Rialto-PoA -> Rialto currency exchange proofs.
+- Rialto-PoA' `Arthur`: signs relay transactions with new Rialto headers;
+- Rialto-PoA' `Bertha`: signs currency exchange transactions.
+
+Following accounts are used when `rialto-millau` bridge is running:
+
+- Millau' `Charlie` signs relay transactions with new Rialto headers;
```

Comment:
There's no need for punctuation at the end of the bullet points (`;` and `.`) - these aren't sentences

Reply:
I don't think we have any strict rule about this defined? I could see Substrate is using punctuation in many places (e.g. [here](https://github.com/paritytech/substrate/tree/master/frame/system), [here](https://github.com/paritytech/substrate/tree/master/frame/transaction-payment), [here](https://github.com/paritytech/substrate/tree/master/frame/recovery)), so I don't think it should be changed.

Reply:
I just read some posts on the punctuation of bullet lists, and the gist I got was that we _may_ use periods at the end of each point here. The reason is because they're (kinda) complete sentences. However, we should _not_ be using semicolons. The examples you gave are clearly full sentences, which is why the have periods at the end.

If we want to be consistent, which is one approach to addressing this, then we should just use periods. However, I don't really want to spend more time writing about punctuation, so I'll let you decide what's best.

Reply:
[That's the](https://english.stackexchange.com/questions/75591/semicolons-at-the-end-of-list-items) first line in the "punctuation at bullet lists semicolon" google results, so I'm going with semicolon.